content/iOS/Practices/iOS-practices.md

```diff
@@ -0,0 +1,118 @@
+# A few iOS practices that can go a long way when maintaining your projects.
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team who writes about the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
+
+
+## Introduction 
+
+Have you ever had to maintain or add features to a project you developed in the past and think: who wrote this and why do they hate me so much? 
+
+I think this is quite normal since having in mind that some projects grow way bigger than we initially expected, and even when you have the best intentions, and you put lots of love and hard work in them, as you mature as a developer you start incorporating more and more practices, and the way you develop changes drastically, making it difficult and annoying to go back to older projects.
```

Comment:
Something bothers me in this part `since having in mind`, I don't know if it is just me. 
I think maybe I would say `I think this is quite normal**,** having in mind that...` (adding a comma and deleting the since) or maybe like this `I think this is quite normal **considering** that...`. I know considering looks like Spanish badly translated but google told me it existed and it was ok to use it 😄

Reply:
I added a comment for this too. I can't wrap my mind around the `since having in mind`. I proposed changing `since having` for `if we have`, but I really like the `considering that` alternative too.